specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json

```diff
@@ -1948,6 +1948,19 @@
         "modelAsString": true
       }
     },
+    "Ipv6CircuitConnectionConfig": {
+      "description": "IPv6 Circuit Connection properties for Global Reach.",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "/125 IP address space to carve out Customer addresses for Global Reach."
+        },
+        "circuitConnectionStatus": {
+          "$ref": "#/definitions/CircuitConnectionStatus",
```

Comment:
Is this something the user can update or is it a read-only property? If read-only, please mark it as such.

Reply:
Marked as read-only